src/services/fileStorage/utils/updateParentDirectories.js

```diff
@@ -0,0 +1,20 @@
+const { FileModel } = require('../model');
+const { courseModel } = require('../../user-group/model');
+const { teamsModel } = require('../../teams/model');
+
+const updateParentDirectories = async (resourceID) => {
+	const file = await FileModel.findById(resourceID).lean().exec();
+	if (file) {
+		const parentID = file.parent;
+		if (parentID || file.refOwnerModel === 'user') {
+			await FileModel.updateOne({ _id: parentID }, { updatedAt: new Date().toISOString() }).exec();
+			await updateParentDirectories(parentID);
```

Comment:
what if parentID is undefined?

Reply:
I updated the code which only runs if parentID is defined without checking for the `refOwnerModel` because this is the default case. Now it should only try to update the file if the ID is defined.